cdk/.nvmrc

```diff
@@ -0,0 +1 @@
+v21
```

Comment:
Could we used a fully qualified version to increase determinism? Would also suggest using Node 20, as it's the LTS version.

Reply:
Done in aceb2be3274a9cc9ae46d96099bf5b700ff3bfbc